test/instance_validity_test.rb

```diff
@@ -93,4 +93,63 @@ def test_assert_inapplicable_for_vroom_if_vehicle_distance
     assert_includes OptimizerWrapper.config[:services][:vroom].inapplicable_solve?(vrp), :assert_no_distance_limitation
     refute_includes OptimizerWrapper.config[:services][:ortools].inapplicable_solve?(vrp), :assert_no_distance_limitation
   end
+
```

Comment:
can you add a functionnal test (in wrapper_test.rb) to check API returns a 400 http status code with duplicated IDs?